pwndbg/commands/search.py

```diff
@@ -105,6 +104,19 @@ def print_search_hit(address) -> None:
     "-e", "--executable", action="store_true", help="Search executable segments only"
 )
 parser.add_argument("-w", "--writable", action="store_true", help="Search writable segments only")
+parser.add_argument(
+    "-i",
+    "--interval",
```

Comment:
Shall we call it `-d --distance`? Or any other ideas?

cc: @gsingh93 @Arusekk @lebr0nli for opinion

Reply:
fwiw to me interval makes sense (which is why I named it that way), as a search interval implies a repetition with a fixed value. I just am not sure it would be obvious to others, especially none-native english speakers. But --distance doesn't imply repetition to me so I think might be even more confusing.

Reply:
Let's do `-s --step`